### DISCUSSION TITLE
How to stop flooding my MQTT with stat/RESULT on every rule command

### DISCUSSION THREAD

**Discussion by odya** [arendst/Tasmota#10957] [Q&A] [answered]
I have a rule that sets **Var1** on every PulseCounter change. It happens nearly once per 2 seconds. And it floods my MQTT sending **RESULT** topic on every **Var1** update.

I searched over all documentation, but I did not find any solution to this. Is there some way to turn off this logging? (I already tried all logs disabling)

**Comment by sfromis** [comment, marked as answer]
That's how it is. Just ignore messages you have no need for.

**Comment by petero-dk** [comment]
Is there a way to propose an option to do this? I want do a simple calculation on a external blink sensor which sometimes blinks twice per second.

**Reply by sfromis** [reply]
There is the topic of "Ideas" here within Github Discussions for such.

In general, new features have a much better chance of implementation if someone works to create a coherent implementation, and submit a PR, instead of "just" asking to fix something which may not even be a "real" problem. I have no guess either way for how likely it would be to be accepted.

My preference for implementing local processing of data would be Berry (available with ESP32), or possibly scripting (also available on ESP8266, may take some learning curve to grok). Rules are more designed for creating simple automations.
